cost-analyzer/templates/cost-analyzer-cluster-role-template.yaml

```diff
@@ -64,6 +64,8 @@ rules:
     verbs:
       - list
       - watch
+      - update
+      - get
```

Comment:
I saw above in this file, there's a "deployments" under `apiGroups: ''` with `get, list, watch` only permissions. What's the difference between that and the `apiGroups: extensions` ?

Reply:
Deployments had API versions: `extensions/v1beta1`, `apps/v1beta1`, and `apps/v1beta2` [until K8s 1.16](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/). Those patch notes include similar info about DaemonSets, StatefulSets, etc. So I think if we need to access deployments (etc.) in K8s versions pre-1.16 we would need the `apiGroups: extensions` permissions.